lib/evss/disability_compensation_form/form526_to_lighthouse_transform.rb

```diff
@@ -267,25 +267,39 @@ def transform_toxic_exposure(toxic_exposure_source) # rubocop:disable Metrics/Me
 
         # create an Array[Requests::MultipleExposures]
         multiple_exposures = []
-        if toxic_exposure_source['gulfWar1990Details'].present?
```

Comment:
I'm sure I'm missing something, but why couldn't we just check the source object here? i.e. check `if gulf_war1990.present?` and if it's not, then skip adding in the details?